es-MX/firefox/whatsnew/whatsnew.ftl

```diff
@@ -2,13 +2,13 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-
 ### URL: https://www-dev.allizom.org/firefox/60.0/whatsnew/all/
 
-whatsnew-page-title = Novedades de { -brand-name-firefox } - más privacidad, más protecciones.
-whatsnew-page-description = Posiciónate en contra de una industria que vende tus datos a terceros. Permanece atento y a salvo en la red con tecnología que te cuida.
-whatsnew-up-to-date-notification = ¡Felicidades! Ahora tienes la última versión de { -brand-name-firefox-browser }.
-whatsnew-out-of-date-notification = Un { -brand-name-firefox-browser } aún más nuevo está disponible. Reiniciar para actualizar.
+whatsnew-page-title = Qué hay de nuevo en { -brand-name-firefox } - Más privacidad, más protecciones.
+whatsnew-page-description = Pronúnciate en contra una industria que vende tus datos a terceros. Ponte listo y mantente seguro en línea con la tecnología que lucha por ti.
+whatsnew-up-to-date-notification = ¡Enhorabuena! Estás utilizando la versión más reciente de { -brand-name-firefox-browser }.
+whatsnew-out-of-date-notification = Está disponible una versión más nueva de { -brand-name-firefox-browser }. Reinicia para actualizar.
+
 # Variables:
 #   $url (url) - link to https://www.mozilla.org/firefox/notes/
-whatsnew-release-notes = Lee las <a href="{ $url }">Notas de versión</a> para saber sobre las novedades de tu navegador { -brand-name-firefox }.
+whatsnew-release-notes = Lee las <a href="{ $url }">Notas de la versión</a> para obtener más información sobre las novedades de su navegador { -brand-name-firefox }.
```

Comment:
Grammar: "la" is appropriate if the version number is specified. Otherwise, it should be removed.
Style: "su" is a regression. es-MX uses the informal "tú", as written in the original translation.
Style: "obtener más información" makes the string longer. I would try to make it shorter to stay closer to the English string length.